### PR TITLE
feat: add simple reporting mechanism to page

### PR DESCRIPTION
**Pull request by maehr**
# Pull request

## Proposed changes

- Add simple reporting mechanism as discussed in https://github.com/Stadt-Geschichte-Basel/stadtgeschichtebasel.ch/issues/58

## Types of changes

- [x] New feature (non-breaking change which adds functionality).
- [ ] Enhancement (non-breaking change which enhances functionality)
- [ ] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[README](./README.md)** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Summary by CodeRabbit

- **New Features**
	- Enhanced the navigation component to generate a mailto link for reporting issues related to the current page, simplifying the process for users to provide feedback and contact information.

### CODE REVIEW

File: src/lib/components/Navigation.svelte [hunk 1]
```diff
@@ -14,6 +14,14 @@
...
 		{ label: 'Über uns', path: '/ueber-uns' }
 	];
+	const currentURL = $page.url.pathname;
+	const mailtoBody = encodeURI(
+		`Ich habe ein Problem auf der Seite entdeckt: ${currentURL}\n` +
+			`Art des Problems: [Bitte angeben: Technischer Fehler, Inhaltlicher Fehler, Tippfehler, Sonstiges]\n` +
+			`Beschreibung des Problems: [Bitte beschreiben Sie das Problem so genau wie möglich.]\n` +
+			`Weitere Anmerkungen: [Optional]\n` +
+			`Kontaktinformationen: [Optional, falls Rückmeldung erwünscht]\n`
+	);
```
[the remark this answers is not in the document]
Comment (maehr) [reply src/lib/components/Navigation.svelte:25]: $page.url.pathname is already encoded as a URI (it is a URL)

**Comment by maehr** [comment]
@mtwente If you could have a look at the design I'd be very happy.

File: src/lib/components/Navigation.svelte [hunk 2]
```diff
@@ -35,4 +48,10 @@
 			</li>
 		{/each}
 	</ul>
+
+	<a
+		class="variant-ghost btn btn-sm mt-4"
```
Comment (mtwente) [inline src/lib/components/Navigation.svelte:53]:
```suggestion
		class="variant-ringed btn btn-sm mt-4"
```

**Review by mtwente — COMMENTED** [review]
Adding the current URL to the email subject and text body does not seem to work properly. When I click the button on one page, e.g. /blog, my email software opens. When I then close the email window, go to another page, e.g. /agenda, and click the button again, the email subject still says "Fehler auf /blog". Is this because I am using an offline preview on my computer?

**Review by koilebeit — APPROVED** [review]

**Pull request merged**